packages/hardhat/tasks/verifyContract.ts

```diff
@@ -0,0 +1,202 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
```

Comment:
Great work on the script. A point of consideration is that maybe we'd want to add this test into the deployment scripts for it to automatically fire off per deployment.

Sometimes it might fail even, and that's okay as the user can just rerun that specific script again.

Reply:
Ok I will add a timeout so that filfox indexes first the new contract

Reply:
@xBalbinus Done!